src/Form/ServerForm.php

```diff
@@ -212,7 +212,14 @@ public function form(array $form, FormStateInterface $formState): array {
       '#description' => $this->t('Security rule: The maximum allowed complexity of a query. Leave empty to set unlimited.'),
     ];
 
-    $debug_flags = $server->get('debug_flag') ?? 0;
+    $debug_flags_array = $server->get('debug_flag');
+    $debug_flags = 0;
+    if (is_array($debug_flags_array)) {
```

Comment:
Thanks, but I think this is not the correct fix. `$server->get('debug_flag')` must always return an integer, right? How did the array get into this property in the first place? We need to prevent that.

Reply:
okily dokily :)